flink-yarn/src/test/java/org/apache/flink/yarn/YarnFileStageTest.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.flink.util.OperatingSystem;
 import org.apache.flink.util.TestLogger;
 
+import com.google.common.collect.Lists;
```

Comment:
Instead of directly using Guava you should use the `flink-shaded-guava` version.

Reply:
Changed.